File: vue-vben-admin/apps/web-naive/src/api/dto/RoleDto.ts
export interface RoleDto {
  id: string  | null | undefined ;    //  主键
  name: string  | null | undefined ;    //  角色名称
  fullName: string  | null | undefined ;    //  角色全名称
  parentId: string  | null | undefined ;    //  父级Id
  remark: string  | null | undefined ;    //  备注
}

File: src/SimpleApp/Simple.AdminApplication/SysMng/Dto/RoleDto.cs
using Simple.Utils.Models.Dto;

namespace Simple.AdminApplication.SysMng.Dto;

public class RoleDto : BaseDto, ITreeDto
{
    /// <summary>
    /// 主键
    /// </summary>
    public Guid Id { get; set; }

    /// <summary>
    /// 角色名称
    /// </summary>
    public string Name { get; set; }

    /// <summary>
    /// 角色全名称
    /// </summary>
    public string FullName { get; set; }

    /// <summary>
    /// 父级Id
    /// </summary>
    public Guid? ParentId { get; set; }

    /// <summary>
    /// 备注
    /// </summary>
    public string Remark { get; set; }
}